test/web3/shouldWorkWithWeb3.ts

```diff
@@ -62,6 +74,22 @@ export function shouldWorkWithWeb3({
           receipt = await web3Box[0].eth.getTransactionReceipt(txResult.hash)
         }
         return receipt
+      },
+      waitSafeTxReceipt: async (txResult: TransactionResult): Promise<any> => {
```

Comment:
Maybe should be a `Promise<Record<string, any> | undefined>`, or even a `Promise<Record<string, any>>` if returning undefined can get changed to throwing an error.